gradle/libs.versions.toml

```diff
@@ -0,0 +1,105 @@
+[versions]
+kotlin-version = "1.6.10"
```

Comment:
may be remove `version` suffix?

Reply:
They are not used directly, only as refs and this file. I'm trying to keep short names for the libs themself